Makefile

```diff
@@ -15,7 +15,7 @@ OS := $(shell uname -s | tr A-Z a-z)
 ARCH := amd64
 ###############################################################################
 # Subcomponent versions:
-GO_BUILD_VER:=latest
+GO_BUILD_VER:=v0.9
```

Comment:
@fasaxc - it looks like `quay.io/calico/go-build:latest` was updated on quay.io earlier today, but the hash doesn't match up with anything else. Do you know what's been in `latest` and how long calicoctl has been building with it?

Reply:
Looks like that's `master`